pkgs/tools/virtualization/mkosi/default.nix

```
@@ -46,7 +46,7 @@ let
     withKernelInstall = true;
   };
 
-  python3pefile = python.withPackages (_: [ pefile ]);
+  pythonWithPefile = python.withPackages (_: [ pefile ]);
```

```suggestion
  pythonWithPefile = python.withPackages (ps: [ ps.pefile ]);
```
Remove pefile from parameters if not used elsewhere.